src/python/zquantum/core/interfaces/backend.py

```diff
@@ -23,6 +23,8 @@ def run_circuit_and_measure(self, circuit:Circuit, **kwargs) -> List[Tuple]:
         Method for executing the circuit and measuring the outcome.
         Args:
             circuit (core.circuit.Circuit): quantum circuit to be executed.
+        Returns:
+            Measurements: object representing the measurements resulting from the circuit
```

Comment:
Actually it should be named "core.measurements.Measurements" I guess ;)

Reply:
Addressed here: 
5309e30